book/generating-proofs/network.md

```diff
@@ -14,34 +14,24 @@ SP1_PROVER=network SP1_PRIVATE_KEY=... cargo run --release
 
 - `SP1_PROVER` should be set to `network` when using the prover network.
 
-- `SP1_PRIVATE_KEY` is your secp256k1 private key for signing messages on the network. The balance of
-  the address corresponding to this private key will be used to pay for the proof request.
+- `SP1_PRIVATE_KEY` is your secp256k1 private key for signing messages on the network. You will need
+  to be using a [whitelisted](#getting-whitelisted) key to use the network.
```

Comment:
This is too complex for users.

Your API key is the private key for the address that's whitelisted on the network! We should give more detail on how this is set up in this network section as well. i.e. Here's the contract on Base and more cc @puma314

Reply:
added more details, also we no longer use the base contract